src/classes/Commands.ts

```diff
@@ -321,6 +325,79 @@ export = class Commands {
         );
     }
 
+    private messageCommand(steamID: SteamID, message: string): void {
+        const isAdmin = this.bot.isAdmin(steamID);
+        const parts = message.split(' ');
+
+        if (process.env.DISABLE_MESSAGES === 'true') {
+            if (isAdmin) {
+                this.bot.sendMessage(
+                    steamID,
+                    'The message command is disabled. Enable it in the config with `DISABLE_MESSAGES=false`.'
+                );
+            } else {
+                this.bot.sendMessage(steamID, 'The owner has disabled messages.');
+            }
+            return;
+        }
+
+        const details = this.bot.friends.getFriend(steamID);
+
+        if (isAdmin) {
+            if (parts.length < 3) {
+                this.bot.sendMessage(
+                    steamID,
+                    'Your syntax is wrong. Here\'s an example: "!message 76561198120070906 Hi"'
+                );
+                return;
+            }
+
+            const recipient = parts[1];
+
+            if (!new SteamID(recipient).isValid()) {
+                this.bot.sendMessage(steamID, '"' + recipient + '" is not a valid steamid.');
+                return;
+            } else if (!this.bot.friends.isFriend(recipient)) {
+                this.bot.sendMessage(steamID, 'I am not friends with the user.');
+                return;
+            }
+
+            const reply = parts[2];
+
+            // Send message to recipient
+            this.bot.sendMessage(recipient, 'Message from ' + (details ? details.player_name : 'admin') + ': ' + reply);
```

Comment:
This shouldn't be `(isAdmin ? isAdmin.player_name : 'admin')` yes?

Reply:
No. You getFriend can return null so "admin" is just a placeholder if the name was not found.

Reply:
ah I see.. alright kinda confused a bit 👌🏻